chart/values.yaml

```diff
@@ -756,3 +756,7 @@ localpv-provisioner:
     enabled: false
   hostpathClass:
     enabled: false
+  # Create a hostpath storage class for mayastor-etcd.
+  createEtcdSc: true
```

Comment:
Do we need separate ones for etcd and loki?

Reply:
Yes we do for case when someone wants to disable loki and might not want this storageclass.